.drone.yml

```diff
@@ -9,7 +9,7 @@ platform:
 
 steps:
   - name: dapper
-    image: golang:1.15.7-buster
+    image: golang:1.16.9-buster
```

Comment:
update go.mod if youre gonna do this: https://github.com/rancher/dapper/blob/master/go.mod#L3

Reply:
done, thanks for catching that.